400 http error using multipart behavior

When i tried to use a separate file to customize the test execution i always get 400 http error, we are using xray cloud, if i do the same using postman multipart works, i also tried adding some other keys like `jiraBaseUrl`, `jiraToken` but i always have the same response

pom.xml
```xml
<plugin>
   <groupId>app.getxray</groupId>
   <artifactId>xray-maven-plugin</artifactId>
   <version>0.7.0</version>
   <configuration>
      <scope>test</scope>
      <clientId>***</clientId>
      <clientSecret>***</clientSecret>
      <cloud>true</cloud>
      <reportFormat>cucumber</reportFormat>
      <testExecInfoJson>src/main/resources/xrayInfo.json</testExecInfoJson>
      <reportFile>${project.build.directory}/cucumber-reports/cucumber-report.json</reportFile>
      <abortOnError>true</abortOnError>
   </configuration>
</plugin>
```

error
```
java.io.IOException: Unexpected HTTP code Response{protocol=http/1.1, code=400, message=Bad Request, url=https://xray.cloud.getxray.app/api/v2/import/execution/cucumber/multipart}
	at app.getxray.xray.XrayResultsImporter.submitMultipartCloud(XrayResultsImporter.java:427)
	at app.getxray.maven.plugin.xray.ImportResultsMojo.execute(ImportResultsMojo.java:215)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:137)
	at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute(MojoExecutor.java:301)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:211)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:165)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:157)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:121)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:81)
	at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:56)
	at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:127)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:294)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:192)
	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:105)
	at org.apache.maven.cli.MavenCli.execute(MavenCli.java:960)
	at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:293)
	at org.apache.maven.cli.MavenCli.main(MavenCli.java:196)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:577)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:282)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:225)
	at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:406)
	at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:347)
[ERROR] Unexpected HTTP code Response{protocol=http/1.1, code=400, message=Bad Request, url=https://xray.cloud.getxray.app/api/v2/import/execution/cucumber/multipart}
java.io.IOException: Unexpected HTTP code Response{protocol=http/1.1, code=400, message=Bad Request, url=https://xray.cloud.getxray.app/api/v2/import/execution/cucumber/multipart}
	at app.getxray.xray.XrayResultsImporter.submitMultipartCloud(XrayResultsImporter.java:427)
	at app.getxray.maven.plugin.xray.ImportResultsMojo.execute(ImportResultsMojo.java:215)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:137)
	at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute(MojoExecutor.java:301)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:211)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:165)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:157)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:121)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:81)
	at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:56)
	at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:127)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:294)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:192)
	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:105)
	at org.apache.maven.cli.MavenCli.execute(MavenCli.java:960)
	at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:293)
	at org.apache.maven.cli.MavenCli.main(MavenCli.java:196)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:577)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:282)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:225)
	at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:406)
	at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:347)
```

xrayInfo.json
```
{
  "fields": {
    "summary": "Test execution, 2023/01/26 09:33:50",
    "issuetype": {
      "id": "issue_type_id"
    },
    "project": {
      "id": "project_id"
    },
    "description": "Automated test execution, Platform: android, Date: 2023/01/26 09:33:50",
    "assignee": {
      "id": "assigned_id"
    }
  }
}
```

command to execute tests
```
 mvn clean test -Dcucumber.filter.tags="@login" xray:import-results
```

i hope you can take a look.
Thanks

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -11,6 +11,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
+## [0.7.1] - 2023-02-02
+
+### Fixed
+
+- handling of multipart requests for null test/testExec info JSON (#54)
+
 ## [0.7.0] - 2022-07-01
 
 ### Added
@@ -84,7 +90,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
-[unreleased]: https://github.com/Xray-App/xray-maven-plugin/compare/0.7.0...HEAD
+[unreleased]: https://github.com/Xray-App/xray-maven-plugin/compare/0.7.1...HEAD
+[0.7.1]: https://github.com/Xray-App/xray-maven-plugin/compare/0.7.1...0.7.0
 [0.7.0]: https://github.com/Xray-App/xray-maven-plugin/compare/0.7.0...0.7.0
 [0.6.0]: https://github.com/Xray-App/xray-maven-plugin/compare/0.5.0...0.6.0
 [0.5.0]: https://github.com/Xray-App/xray-maven-plugin/compare/0.4.0...0.5.0

--- a/pom.xml
+++ b/pom.xml
@@ -3,7 +3,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>app.getxray</groupId>
   <artifactId>xray-maven-plugin</artifactId>
-  <version>0.7.0</version>
+  <version>0.7.1</version>
   <packaging>maven-plugin</packaging>
   <name>xray-maven-plugin</name>
   <description>Maven plugin for interacting with Xray (server/datacenter and cloud), used in CI/CD for assisting in test automation flows, such as reporting test results back to Xray and, by consequence, Jira.</description>

--- a/src/main/java/app/getxray/maven/plugin/xray/ImportResultsMojo.java
+++ b/src/main/java/app/getxray/maven/plugin/xray/ImportResultsMojo.java
@@ -179,7 +179,7 @@ public class ImportResultsMojo extends AbstractMojo {
                         .withTimeout(timeout)
                         .withLogger(getLog())
                         .withVerbose(verbose);
-                    if (testInfoJson==null  && testExecInfoJson==null) {       
+                    if (testInfoJson==null  && testExecInfoJson==null) {
                         if (XrayResultsImporter.XRAY_FORMAT.equals(reportFormat) || XrayResultsImporter.CUCUMBER_FORMAT.equals(reportFormat) || XrayResultsImporter.BEHAVE_FORMAT.equals(reportFormat)) {
                             xrayImporter = xrayImporterBuilder.build();
                         } else {
@@ -200,8 +200,6 @@ public class ImportResultsMojo extends AbstractMojo {
                             } else {
                                 abortWithError("file doesnt exist: " + testInfoJson);
                             }
-                        } else {
-                            testInfo = new JSONObject();
                         }
                         if (testExecInfoJson != null) {
                             if ((new File(testExecInfoJson)).isFile()) {
@@ -209,8 +207,6 @@ public class ImportResultsMojo extends AbstractMojo {
                             } else {
                                 abortWithError("file doesnt exist: " + testExecInfoJson);
                             }
-                        } else {
-                            testExecInfo = new JSONObject();
                         }
                         response = xrayImporterBuilder.build().submitMultipartCloud(reportFormat, reportFile, testExecInfo, testInfo);                  
                     }
@@ -257,8 +253,6 @@ public class ImportResultsMojo extends AbstractMojo {
                             } else {
                                 abortWithError("file doesnt exist: " + testInfoJson);
                             }
-                        } else {
-                            testInfo = new JSONObject();
                         }
                         if (testExecInfoJson != null) {
                             if ((new File(testExecInfoJson)).isFile()) {
@@ -266,8 +260,6 @@ public class ImportResultsMojo extends AbstractMojo {
                             } else {
                                 abortWithError("file doesnt exist: " + testExecInfoJson);
                             }
-                        } else {
-                            testExecInfo = new JSONObject();
                         }
                         response = xrayImporterBuilder.build().submitMultipartServerDC(reportFormat, reportFile, testExecInfo, testInfo);                  
                     }

